python_modules/dagster/dagster/generate/generate.py

```diff
@@ -59,7 +61,13 @@ def generate_new_project(path: str):
                 # Jinja template names must use the POSIX path separator "/".
                 template_name = src_relative_file_path.replace(os.sep, posixpath.sep)
                 template = env.get_template(name=template_name)
-                f.write(template.render(repo_name=repo_name))
+                f.write(
+                    template.render(
+                        repo_name=repo_name,
+                        dagster_version=dagster_version,
+                        dagit_version=dagit_version,
```

Comment:
cc @catherinewu, @sryza thoughts on enforcing dagit and dagster version to be the same here?
```suggestion
                        dagit_version=dagster_version,
```

Reply:
For now, we need both versions to be the same (we don't explicitly support running different versions of Dagster and Dagit with each other), so yes we should do this.